ios/BT/ExposureManager.swift

```diff
@@ -440,9 +441,8 @@ final class ExposureManager: NSObject {
         btSecureStorage.storeExposures(newExposures)
         completionHandler(.success(processedFileCount))
       case let .failure(error):
-        let exposureError = ExposureError.default(error.localizedDescription)
         btSecureStorage.exposureDetectionErrorLocalizedDescription = error.localizedDescription
-        completionHandler(.failure(exposureError))
+        completionHandler(.failure(error))
```

Comment:
@mattThousand  we noticed a bug where the API ENError was being converted to our code's 'ExposureError' and losing all of the enum info from ENError.  This was our fix, could you give it a double check?

Reply:
👍